server/app/service/ScalaTestRunner.scala

```diff
@@ -26,7 +26,7 @@ object ScalaTestRunner {
   /**
    * Runs suite loaded in runtime with dynamic solution
    */
-  def execSuite(solution: String, suiteClass: Class[Suite], solutionTrait: Class[AnyRef]): String =
+  def tryExecSuite(solution: String, suiteClass: Class[Suite], solutionTrait: Class[AnyRef]): String =
```

Comment:
let's wrap with tryExec other methods below and rename this one back to execSuite to be consistent with other, because every API method may catch exception when we run reflection instance creation, so that each method should have tryExec inside. As already discussed these two methods are overloaded and they do the same thing. So that they better equal names